File: core/src/main/java/de/qabel/core/StringUtils.java
package de.qabel.core;

import java.util.List;

public class StringUtils {
    public static String join(String separator, String[] parts) {
        return org.apache.commons.lang3.StringUtils.join(parts, separator);
    }
    public static <T> String join(String separator, List<T> parts) {
        return org.apache.commons.lang3.StringUtils.join(parts, separator);
    }
}

File: core/src/main/java/de/qabel/core/StringUtils.kt
package de.qabel.core

object StringUtils {
    fun join(separator: String, parts: Array<String>): String {
        return org.apache.commons.lang3.StringUtils.join(parts, separator)
    }

    fun <T> join(separator: String, parts: List<T>): String {
        return org.apache.commons.lang3.StringUtils.join(parts, separator)
    }
}
